lib/openstax/aws/git_helper.rb

```diff
@@ -8,8 +8,11 @@ def self.sha_for_branch_name(org_slash_repo:, branch:)
       ::Git.ls_remote("https://github.com/#{org_slash_repo}")["branches"][branch][:sha]
     end
 
-    def self.file_content_at_sha(org_slash_repo:, sha:, path:)
-      location = "https://raw.githubusercontent.com/#{org_slash_repo}/#{sha}/#{path}"
+    def self.file_content_at_sha(org_slash_repo:, sha:, path:, github_token:)
```

Comment:
Not sure if some repos call this method directly, so it may be a good idea to use `github_token: nil` or similar to make it optional.